docs/source/api/graphstorm.inference.rst

```diff
@@ -13,7 +13,7 @@ graphstorm.inferrer
 .. autosummary::
     :toctree: ../generated/
     :nosignatures:
-    :template: classtemplate.rst
+    :template: pipelinetemplate.rst
```

Comment:
GSInferrer does not have save_model, remove_saved_model, save_topk_models, get_best_model_path, restore_model, etc.

Reply:
the pipelinetemplate is share by both trainers and inferrers. If there are some methods missing, the Sphinx complior will raise Warnings, but just ignore those missing methods.